user/enterprise/custom-queues.md

```diff
@@ -0,0 +1,89 @@
+
+  - `language` - defines the build environment based on the chosen language of the project†
+  - `group` - mostly semantic, for defining 'groups' of environments†
+  - `owner` - either be an org or a user
+  - `slug` - a repository, in the form: `org/repo` or `user/repo`
+
+
+† `language` and `group` selectors must be specified in the `.travis.yml`. See [the example](#Advanced-Configuration-YAML-Example) for more details. 
+
+Note: We do not recommend using `dist` and `os` for these selectors. These two have some of their own routing processes built-in and may not entirely behave as intended. 
+
+All selectors should be given as a list in the "Advanced Configuration YAML" in the format:
+```yaml
+production:
+  queues:
+  - queue: name
+    selector: value
+  - queue: another.queue
+    selector: different_value
+    selector: something_else
+```
+see the [example](#Advanced-Configuration-YAML-Example) for details on syntax. Click "Save" on the Management Console Settings when you are ready. Travis CI Enterprise will restart, with your new queue settings.
+
+### Advanced Configuration YAML Example 
+
+The syntax for the **Advanced Configuration YAML** field is very important. Improper syntax will result in builds being routed to defaults, usually a `builds.linux` queue (if this has not been modified on your installation.) Syntax for a custom queue looks as follows: 
+
+```yaml
+production:
+  queues:
+  - queue: builds.ruby
+    owner: travis-ci
+    language: ruby
+    group: enterprise
+  - queue: builds.python
+    owner: acnagy
+    language: python
+  - queue: dockercluster
+    services:
+    - docker
+  - queue: legacy
+    group: legacy
+  - queue: docs
+    slug: 'travis-ci/docs-travis-ci-com' 
+```
+
+Although selectors that indicate ownership (such as `owner` and `slug`) do not need to be set in the `.travis.yml`, other selectors must be. For example, to build an `enterprise`, Ruby project owned by the `travis-ci` organization, a `.travis.yml` would need to look as follows:
+
+```yaml
+group: enterprise
+language: ruby
+# rest of the yaml, per standard spec
+```
+{: data-file=".travis.yml"}
+
+but, to build this repo, [`travis-ci/docs-travis-ci.com`](https://github.com/travis-ci/docs-travis-ci-com), no special configuration would be required.
+
+## Define Custom Queues Settings on The Workers
+
+To allocate a worker to a particular queue, the `QUEUE_NAME` variable must be set in the worker config, which is located at `etc/default/travis-worker`. Update the environment variable to match the queue name specified in your [custom queue configuration yaml](#Advanced-Configuration-YAML-Example). Then restart the worker with: 
```

Comment:
must be set  as above

Reply:
I'm not sure this one is referenced above? I'll go with "must be defined as indicated above"

Reply:
Yeah, reword to "set the name in ..." rather than "must be set by "...